examples/Console/TestJaegerExporter.cs

```diff
@@ -42,6 +44,24 @@ internal static object RunWithActivity(string host, int port)
                     {
                         o.AgentHost = host;
                         o.AgentPort = port;
+
+                        // Examples for the rest of the options, defaults unless otherwise specified
+                        o.MaxPayloadSizeInBytes = 4096;
+                        o.ProcessTags = new Dictionary<string, object>
```

Comment:
nit: Might be slightly more readable using index initializer syntax:
```csharp
                        o.ProcessTags = new Dictionary<string, object>
                        {
                            ["myKey1"] = "myVal1",
                            ["myKey2"] = "myVal2",
                        };
```

Reply:
@CodeBlanch  - Cool, I made the change, thanks for the review!